mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -526,7 +526,12 @@
             }
             controls.setPlayState(!play)
         },
-
+        
+        doStop: function () {
+                toast('Stopping current track...')
+                mopidy.playback.stop()
```

Comment:
The indentation on this line doesn't match the indentation in other functions on either side of this function.